### DISCUSSION TITLE
CORS Policy Blockage

### DISCUSSION THREAD

**Discussion by memdigital** [katspaugh/wavesurfer.js#3651] [Q&A] [answered]
Hi All,

I had a lovely setup on my new site using Wavesurfer, but somehow, and I literally have no clue about this side of Javascript, the unpkg.com CDN link has been blocked by CORS Policy, No Access-Control-Allow-Origin - as it say sin the console.

This is the first time I am seeing this.

Any idea how I can solve this issue?

**Comment by katspaugh** [comment, marked as answer]
Unkg seems to be down at the moment, see https://github.com/katspaugh/wavesurfer.js/issues/3650

**Reply by memdigital** [reply]
Thanks kindly, it appears to have sprung back to life, but being blocked still within my preview page mode.
